pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -223,12 +221,9 @@ stdenv.mkDerivation rec {
     "--enable-default-toolkit=${toolkitValue}"
     "--enable-js-shell"
     "--enable-necko-wifi"
-    "--enable-startup-notification"
     "--enable-system-ffi"
     "--enable-system-pixman"
-    "--enable-system-sqlite"
 
-    "--disable-gconf"
```

Comment:
Can we mention somewhere why these options were removed? (in the commit message perhaps)

Reply:
Updated the commit message.

Reply:
`-icu` was there twice.  For the rest I can't see them referenced in the output, so I assume the flags being unsupported means that thunderbird now always bundles those dependencies :-/  I expect we should remove them from build inputs (some time).